test/controllers/import.test.js

```diff
@@ -158,6 +158,9 @@ describe('ImportController tests', () => {
           enableJavaScript: 'true',
         },
         importScript: 'QW5kIGV2',
+        customHeaders: {
+          Authorization: 'Bearer aXsPb3183G',
```

Comment:
Please add an assertion (or if needed, new test) to verify that the headers are present in the SQS message.